### PR TITLE
A new advanced guide for handling pandas in output

### PR DESCRIPTION
**Pull request by harishmohanraj**

### CODE REVIEW

File: nbs/guides/Automatically_Generate_Docstrings.ipynb [hunk 1]
```diff
@@ -5,7 +5,7 @@
    "id": "d42e50e8",
    "metadata": {},
    "source": [
-    "# Auto generate docstrings"
+    "# Automatically generate docstrings"
```
Comment (davorrunje) [inline nbs/guides/Automatically_Generate_Docstrings.ipynb:8]: auto-generating...

File: nbs/guides/Setting_up_social_cards.ipynb [hunk 2]
```diff
@@ -27,9 +32,10 @@
     "https://opengraph.githubassets.com/some-random-stuff-to-refresh-or-timestamp/{user}/nbdev_mkdocs_tutorial\n",
     "```\n",
     "\n",
-    "#### Customization\n",
+    "## Generating new social cards\n",
+    "\n",
     "\n",
-    "To customize the social share image for your project, use the `nbdev_mkdocs social-image generate` CLI command. To view the current configuration options, run the following command in the terminal.\n",
+    "To customize and generate a new social share image for your project, you can use the `nbdev_mkdocs social-image generate` CLI command. To view the current configuration options, run the following command in the terminal.\n",
```
Comment (davorrunje) [inline nbs/guides/Setting_up_social_cards.ipynb:38]: For customizing and generating a new social share image for your project, you can use the `nbdev_mkdocs social-image generate` CLI command.

**Review by davorrunje — CHANGES_REQUESTED** [review]
auto-generating...

File: nbs/guides/Automatically_Generate_Docstrings.ipynb (hunk 1 above)
Comment (harishmohanraj) [reply nbs/guides/Automatically_Generate_Docstrings.ipynb]: Fixed

File: nbs/guides/Setting_up_social_cards.ipynb (hunk 2 above)
Comment (harishmohanraj) [reply nbs/guides/Setting_up_social_cards.ipynb]: Fixed

**Review by davorrunje — APPROVED** [review]

**Pull request merged**